pkg/util/installinfo/install_info.go

```diff
@@ -22,6 +26,13 @@ import (
 	configUtils "github.com/DataDog/datadog-agent/pkg/config/utils"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/datadog-agent/pkg/version"
+	"github.com/google/uuid"
+)
+
+var (
+	configDir       = "/etc/datadog-agent"
+	installInfoFile = filepath.Join(configDir, "install_info")
+	installSigFile  = filepath.Join(configDir, "install.json")
```

Comment:
This will not work on Windows. We need to either update the path to be generic or have this in a dedicated file only compiled on Unix